tsconfig.json

```diff
@@ -13,7 +13,8 @@
     "noUnusedLocals": true,
     "preserveWatchOutput": true,
     "resolveJsonModule": true,
-		"strict": true,
+	"strict": true,
+    "skipLibCheck": true,
```

Comment:
We should be able to drop `skipLibCheck` once https://github.com/jupyterlab/jupyterlab/pull/17141 is in and a new alpha is released; we could probably just merge and release for testing already though.

Reply:
Curious if we could double check the TypeScript build also fails for this PR if we remove `skipLibCheck`? 

Otherwise we could indeed keep it for now and remove it when doing the next update.

Reply:
It was failing for me locally. Let me fix it by bumping the TypeScript version here instead.

Reply:
Yes, linking to https://github.com/jupyterlab/jupyterlab/pull/17141#issuecomment-2574846099 for more info.

Reply:
Done in e2c87ad8ab92f88f61fb23125077b16e19c2ed94.